src/App.tsx

```diff
@@ -1,19 +1,81 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Footer } from './components/Footer';
+import { NewTodoForm } from './components/NewTodoForm';
+import { TodoList } from './components/TodoList';
+import { FilterTypes } from './types/Filter';
+import { Todo } from './types/Todo';
+
+export function getFilteredTodo(
+  todos: Todo[],
+  selectedTab: FilterTypes,
+) {
+  const filterByType = todos.filter((todo) => {
+    switch (selectedTab.id) {
+      case 'active':
+        return !todo.completed;
+
+      case 'completed':
+        return todo.completed;
+
+      default:
+        return todo;
+    }
+  });
+
+  return filterByType;
+}
```

Comment:
Greate folder helper, file with name of function in it and import it where you need :)